### ISSUE TITLE
Deposition screens too large in Safari browser

### ISSUE THREAD

**Issue by perzeus** [uwbmrb/BMRBdep#165]
When I start a deposition on my Safari browser (Mac OS X Mojave), the screen elements are large, like if I am seeing the page in a phone or tablet. I have to expand the browser window beyond full HD size (my screen is 4K-resolution) to be able to see all columns in the "contact person" entry table.

This does not happen on Firefox, Chrome or Opera.

It may be that Safari is recognized as a mobile (phone or tablet) browser. It happens also when going to some web sites, like PC Magazine: I always get served the "mobile" version of the web page. I believe this is really a Safari issue, not a BMRBdep issue, but I report it in case it can be handled in some way.

**Comment by perzeus** [comment]
I need to add that the forms right-hand borders are missing in Safari.

**Comment by jonwedell** [comment]
Interesting. Are you able to provide a screenshot showing what it looks like when it is below full HD size?

**Comment by jonwedell** [comment]
Also, is this just on the 4k monitor, or on the laptop screen as well?

**Comment by perzeus** [comment]
![Screen Shot 2019-12-18 at 11 58 01 AM](https://user-images.githubusercontent.com/10900087/71110920-da9c4080-218d-11ea-9a0e-97083a597add.png)

**Comment by perzeus** [comment]
This is in comparison with a smaller Firefox screen
![Screen Shot 2019-12-18 at 11 58 52 AM](https://user-images.githubusercontent.com/10900087/71110953-ee47a700-218d-11ea-9d94-98c0bec4bf99.png)

**Comment by jonwedell** [comment]
Interesting. There is a horizontal scroll bar in the Contact person loop to enable viewing the fields that you can't see with the smaller browser. I've played around with this a decent bit before, and wasn't able to come up with a better solution than the scrolling, without compacting some of the tags to be far too narrow.

The other possible alternative would be to break the loop up into two rows on screen per one row in the actual loop, but that would be a lot of work and present a lot of it's own problems.

**Comment by perzeus** [comment]
As you can see, the main problem is that the fonts are too large by default on Safari (I also checked this on the laptop screen: same problem). If I use "zoom out" (Command--) then I can get it to fit HD size and even a bit lower.

**Comment by perzeus** [comment]
I don't think you should break the loop. The user can zoom out if necessary. As I said, this looks like a Safari-only problem.

**Comment by jonwedell** [comment]
> fonts are too large by default on Safari

This is probably straightforward to resolve.
